carbonserver/tox.ini

```diff
@@ -8,7 +8,7 @@ deps =
 
 commands =
     pip install -e .
-    python -m pytest
+    python -m pytest -vv
```

Comment:
Are you sure you want to commit this ?

Reply:
I guess it will be useful when we will add those tests to the CI.